### PR TITLE
Unused code cleanup

### PR DESCRIPTION
**Pull request by ngreenwald**
- Removed code that was no longer being used for pipeline for personal sanity as well as to get build to start passing. 

- I removed the entire "deprecated" folder

- I removed references to K.image_format() as suggested by Will in #45, and instead added in default arguments to specify the data format, so that tensorflow is no longer a dependency

- I removed the majority of the utils functions, which were copied over from deepcell-tf but never actually used. 

- I removed almost all of the test functions, since they were likewise copied over from deepcell-tf, and test functions that are not currently used

- I kept the few utils functions that are currently used in the pipeline. However, even those that are used will likely not be used anymore once we merge the caliban_pipeline branch in. 

- There are a few test functions which I didn't delete. However, I commented them out because they use the tf.Test class in order to do testing. The alternative to this hack would be to a) inherit from a different testing library or b) refactor them to not need an test class. However, given that they're not testing anything relevant to the current pipeline and won't be around for much longer, I don't think this will be a long-term problem

### CODE REVIEW

**Comment by ngreenwald** [comment]
closes #40

File: .coveragerc [hunk 1]
```diff
@@ -9,7 +9,7 @@ exclude_lines =
     raise NotImplementedError
 
 ignore_errors = True
-fail_under = 50
+fail_under = 01
```
Comment (willgraf) [inline .coveragerc]: Why do this if we're already turning off pep8?
Let's leave the `fail_under` the same, and just remove the `--cov` from the travis testing for now.

File: requirements.txt [hunk 2]
```diff
@@ -3,7 +3,6 @@ numpy>=1.14.5,<2
 scipy>=1.1.0,<2
 scikit-image>=0.14.2,<1
 scikit-learn>=0.19.1,<1
-tensorflow==1.11.0
```
Comment (willgraf) [inline requirements.txt]: 😃

**Comment by ngreenwald** [comment]
Got it. Maybe @MekWarrior or @geneva-miller could comment on why these were added in the first place?

My understanding of the long term plan for the toolbox is to preprocess data prior to annotation. Based on that, I don't see why the deepcell utils would be useful here, but I could definitely be missing something

File: .coveragerc (hunk 1 above)
Comment (ngreenwald) [reply .coveragerc]: Good idea

**Comment by ngreenwald** [comment]
Alright, I switched the coverage default back to 50 and removed coverage requirements from travis

**Comment by willgraf** [comment]
I am approving this PR, but I am also requesting @geneva-miller approve it as well, since she is a power user of this project.

**Pull request merged**